src/DevChatter.Bot.Infra.Ef/AppDataContext.cs

```diff
@@ -18,6 +18,7 @@ public class AppDataContext : DbContext
         public DbSet<ScheduleEntity> ScheduleEntities { get; set; }
         public DbSet<CommandUsageEntity> CommandUsages { get; set; }
         public DbSet<QuizQuestion> QuizQuestions { get; set; }
+        public DbSet<RouletteSettingsEntity> RouletteSettings { get; set; }
```

Comment:
Seems weird to have a table with one row. Perhaps a CommandSettings table with a key-value setup?

Reply:
Perhaps something like `CommandSettings` and have this structure

- Id
- CommandNameFull
- Key
- Value